docs/docker-compose/config/konnectd.cfg

```diff
@@ -0,0 +1,2 @@
+oidc_issuer_identifier=https://kopano.demo:9443
+insecure = yes
```

Comment:
Uhm - why is this needed?

Reply:
This is just a file for demonstration in the documentation